packages/modules/web-viewer-react/package.json

```diff
@@ -39,6 +39,7 @@
     "@itwin/core-quantity": "^3.2.0",
     "@itwin/core-react": "^3.2.0",
     "@itwin/core-telemetry": "^3.2.0",
+    "@itwin/imodels-client-management": "^1.0.1",
```

Comment:
same thing about possibily making this a direct dep

Reply:
Moved it to `dependencies`.